src/units/attack_type.cpp

```diff
@@ -146,18 +148,14 @@ static bool matches_simple_filter(const attack_type & attack, const config & fil
 	if ( !filter_name.empty() && std::find(filter_name.begin(), filter_name.end(), attack.id()) == filter_name.end() )
 		return false;
 
-	if ( !filter_type.empty() && std::find(filter_type.begin(), filter_type.end(), attack.type()) == filter_type.end() )
-		return false;
-
-	if(!filter_alternative_type.empty() || !filter_replacement_type.empty()){
-		std::pair<std::string, std::string> damage_type = attack.damage_type();
-		if(!filter_replacement_type.empty()){
-			if ( std::find(filter_replacement_type.begin(), filter_replacement_type.end(), damage_type.first) == filter_replacement_type.end() ){
+	if (!filter_type.empty()){
+		if(tag_name == "damage_type"){
```

Comment:
Add a comment here explaining why damage_type is handled differently.

Reply:
it's done